modules/javascript/pages/react.adoc

```diff
@@ -1,56 +1,48 @@
 = React Native
 :page-partials:
+:page-edition: Public Preview
 :page-toclevels: 2@
-:description: Using Couchbase Lite with Javascript applications
+:description: The React Native plugin for Couchbase Lite is now in public preview. This is an open-source, community-supported project developed by Couchbase.
 
 include::partial$_set_page_context_for_javascript.adoc[]
 
-:param-abstract: If you are building mobile or desktop applications using web technologies such as JavaScript/HTML/CSS, there are options for using Couchbase Lite as an embedded database within your app.
-:param-related: xref:ionic.adoc[] | xref:cordova.adoc[]
-include::{root-partials}_show_page_header_block.adoc[]
+== High-Level Overview
+Couchbase Lite Enterprise Edition for React Native is a Native Module implementation using TypeScript, with feature parity to Couchbase Lite implementations on other platforms.
 
+More information on React Native Native Modules can be found https://cbl-reactnative.dev/[here].
 
-:tutorial: https://github.com/couchbaselabs/userprofile-couchbase-mobile-reactnative/blob/main/README.md
+== Important Notes
 
-:docs: https://reactnative.dev/docs/native-modules-intro
+* Platform Support - This plugin currently only supports iOS and Android platforms. Web, Windows, and macOS are not currently supported.
 
-:ref_implementation: https://github.com/couchbaselabs/couchbase-lite-react-native-module/
+* Enterprise License Required - The Native Module is based on supporting Couchbase Lite Enterprise for iOS and Android. It is not compatible with Couchbase Lite Community Edition, and a license is required to use the Enterprise edition.
 
-NOTE: Couchbase Lite for React Native is a community-driven project without official support from Couchbase.
+* Community-Supported -  This open-source plugin is Couchbase community supported and maintained, and it requires an Enterprise Edition Couchbase Lite license. 
+While it is not officially supported by Couchbase, any issues specific to Couchbase Lite are covered under our existing support policy.
```

Comment:
This sounds like it *is* supported.

Reply:
@ElliotFrancisHunter  This section is not correct and it's quite confusing.  In general. please apply all the comments on Ionic page to the React Native page

Reply:
Done